content/css/concepts/filter-functions/terms/opacity/opacity.md

```diff
@@ -38,3 +38,5 @@ Set the opacity of our element to `50%`:
   filter: opacity(50%);
 }
 ```
+
+![css-filterfunction-opacity-example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-filterfunction-opacity-example.png)
```

Comment:
From your output remove the text below the images reading `Image with opacity filter` and `Image without opacity filter`.

```suggestion
![css-filterfunction-opacity-example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-filterfunction-opacity-example.png)

[add the description here]
```
Then add a sentence below the output that states which image is which.